roles/hosted_engine_setup/tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -54,16 +54,24 @@
       - "OVESETUP_DWH_DB/password=str:{{ he_dwh_db_password }}"
     when: he_dwh_db_password is defined
   - name: Import OpenSCAP task
-    import_tasks: apply_openscap_profile.yml
+    import_tasks: ../apply_openscap_profile.yml
```

Comment:
Why do we need to import from "../"? And is it only in this file? Perhaps worth a separate patch.

Reply:
This is handled in https://github.com/oVirt/ovirt-ansible-collection/pull/271, so we need to rebase it.

Reply:
I do not see that you rebased. Not sure about your exact plans nor whether github makes it easy to rebase a PR on top of another PR.